lib/app/doc/app-nit-lifecycle.tex

```diff
@@ -0,0 +1,19 @@
+\documentclass[tikz]{standalone}
+
+\usetikzlibrary{automata,positioning}
+
+\begin{document}
+\begin{tikzpicture}[->,shorten >= 1pt,node distance=3cm,auto]
+	\node[state] (off)   {off};
```

Comment:
Yeah, that would be useful... I'm working on a way to document platform specific callbacks and link them to the _app.nit_ equivalent, and point out the ones that are not used. Everything in one graph would be too much, but maybe I could show the Android and iOS life-cycle graphs and add the _app.nit_ callbacks on the right edges. For now I have a comparison of the app life-cycle under Android, iOS and _app.nit_: http://xymus.net/pub/lifecycles.pdf, it could be extended for the doc.